README.md

```diff
@@ -171,6 +171,11 @@ const response = await dwolla.post("customers", {
 * If you have any feedback, please reach out to us on [our forums](https://discuss.dwolla.com/) or by [creating a GitHub issue](https://github.com/Dwolla/dwolla-v2-node/issues/new).
 * If you would like to contribute to this library, [bug reports](https://github.com/Dwolla/dwolla-v2-node/issues) and [pull requests](https://github.com/Dwolla/dwolla-v2-node/pulls) are always appreciated!
 
+## Docker
+
+If you prefer to use Docker to run dwolla-v2-node locally, a Dockerfile is included at the root directory.
+Follow these instructions from [Docker's website](https://docs.docker.com/build/hellobuild/) to create a Docker image from the Dockerfile, and run it.
+
```

Comment:
```suggestion
```

Reply:
Added docker information under the Community contribution section. I think it's more relevant there, but if not, feel free to disregard!